src/app/page.tsx

```diff
@@ -13,14 +14,20 @@ export default function Home() {
   const [searchTerm, setSeachTerm] = useState("");
   const [results, setResults] = useState<Result[]>([]);
 
+  const debounceDelay = 500;
+  const debouncedSearchTerm = useDebounce(searchTerm, debounceDelay);
```

Comment:
Do you now need to do this? or is it working that out for iteself?
```suggestion
  const debouncedSearchTerm = useDebounce<T>(searchTerm, debounceDelay);
```

Reply:
@matt-greenfield-15 typescript isn't moaning so I guess it knows whats up